worklets/Overview.bs

```diff
@@ -158,21 +169,25 @@ Note:
 ### The event loop ### {#the-event-loop}
 
 Each {{WorkletGlobalScope}} object has a distinct <a>event loop</a>. This <a>event loop</a> has no
-associated <a>browsing context</a>, and only its <a>microtask queue</a> is used (all other <a>task
-queues</a> are not used). The <a>event loop</a> is created by the <a>create a
+associated <a>browsing context</a>. Only tasks associated with the author invoking
+{{Worklet/import()}} use the <a>event loop</a>, in addition to the <a>microtask queue</a> when the
+user-agent invokes callbacks. The <a>event loop</a> is created by the <a>create a
 WorkletGlobalScope</a> algorithm.
```

Comment:
Shouldn't the user-agent callbacks also use tasks? Otherwise promise code that resolves within the same task can get intertwined with callbacks coming from elsewhere leading to weird inconsistencies. I doubt that's how anyone would actually implement this.